src/components/notes/editor.jsx

```diff
@@ -168,7 +157,7 @@ export default function Editor(props) {
                 collabProvider.serviceClient.cozyClient.getStackClient().uri
               }
               isPublic={isPublic}
-              file={doc.file}
+              file={file}
```

Comment:
Should we render all these components only if `file` is defined? Having to check `if(file)` everywhere (back_from_editing, header_menu, headernote...) doesn't seem right.

Reply:
You are right its remainder from a previous development iteration

Reply:
I refactored this part to remove the conditions